smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -0,0 +1,201 @@
+// ************************************************************
```

Comment:
this isn't a copy of the latest from Veer.  It seems like the vanilla one from swerve drive specialties.  Is there a reason not to start from the latest on Veer and modify from there?

Reply:
so this is a copy of what is in the main branch ... do you know the branch I should get it out of?